modules/rhel-adding-more-nodes.adoc

```diff
@@ -58,10 +58,17 @@ In this example, the `mycluster-rhel7-0.example.com` and
 `mycluster-rhel7-1.example.com` machines are in the cluster and you add the
 `mycluster-rhel7-2.example.com` and `mycluster-rhel7-3.example.com` machines.
 
-. Run the scaleup playbook:
+. Navigate to the Ansible playbook directory:
 +
+[source,terminal]
 ----
 $ cd /usr/share/ansible/openshift-ansible
+----
+
+. Run the scaleup playbook:
++
+[source,terminal]
+----
```

Comment:
Do we want a [source,terminal] after Step 3 (_Add a [new_workers] section_)?

Reply:
Since an Ansible inventory file is kind of a wildcard when it comes to format, I opted to leave it as generic source.